### DISCUSSION TITLE
MSCV%40CMU/

### DISCUSSION THREAD

**Discussion by giscus** [opencsapp/opencsapp.github.io#221] [Announcements]
# MSCV%40CMU/

None

https://opencs.app/MSCV%40CMU/

**Comment by 729593736** [comment]
想请问一下mscv要gre成绩吗？btw，cv顶会非一作有用不

**Reply by RainJamesY** [reply]
scs下的GRE应该都是required

**Comment by Skylar-xty** [comment]
这个和mrsd比怎么样？

**Reply by RainJamesY** [reply]
mrsd会更偏robo一点 会上一些硬件的课程；cv更软

**Reply by jimlue530** [reply]
2023 MRSD錄取率35% MSCV錄取率25%

**Comment by Eric-zhang-1111** [comment]
cmu的funding有被砍吗
